tools/windows-node-installer/log/log.go

```diff
@@ -0,0 +1,20 @@
+package log
+
+import (
+	"github.com/go-logr/zapr"
+	"go.uber.org/zap"
+	logger "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+// SetLogLevel sets the level of log for display in command line using zapcore.Levels ("debug", "info", "warn",
+// "error", "dpanic", "panic", and "fatal") returns an error if input log level is not understandable.
+func SetLogLevel(logLevel []byte) error {
```

Comment:
nit: For UX puposes it might be better to have this take in a string, and then convert to a byte array? That way everytime we use this function we wont have to do `[]byte(foo)`

Reply:
I just copied this method from other place. I haven't give much thought into this. I'll look more into this perhaps in a separate PR.